src/QMCDrivers/VMC/VMCUpdateReny.cpp

```diff
@@ -0,0 +1,569 @@
+//////////////////////////////////////////////////////////////////////////////////////
```

Comment:
No idea, I guess this was fully implemented and may need very minor fix like RMC but need tests.